homeassistant/components/zwave_js/climate.py

```diff
@@ -336,9 +336,11 @@ def device_state_attributes(self) -> Optional[Dict[str, str]]:
     def supported_features(self) -> int:
         """Return the list of supported features."""
         support = SUPPORT_PRESET_MODE
-        if len(self._current_mode_setpoint_enums) == 1:
+        # If any setpoint value exists, we can assume temperature
```

Comment:
Is supported features something we could determine in the constructor instead of having to constantly calculate it?

Reply:
yep that's a good idea